aicsimageio/writers/bfio_writers.py

```diff
@@ -0,0 +1,153 @@
+            The array of data to store. Data arrays must have 2 to 5 dimensions. Data
+            should have shape [T,C,Z,Y,X], where interstitial empty dimensions must be
+            present. For example, a 3-channel image that is 1024x1024 must at least have
+            shape [3,1,1024,1024]. Image with shape [3,1024,1024] will be interpreted as
+            an image with 3 z-slices. Sample dimension is not supported.
+            If the data is a Dask array, chunked writing will be performed
+        uri: types.PathLike
+            The URI or local path for where to save the data.
+            Note: OmeTiledTiffWriter can only write to local file systems.
+        ome_xml: Optional[Union[str, OME]]
+            Provided OME metadata. The metadata can be an xml string or an OME object
+            from ome-types. A provided ome_xml will override any other provided
+            metadata arguments.
+            Default: None
+            The passed-in metadata will be validated against current OME_XML schema and
+            raise exception if invalid.
+            The ome_xml will also be compared against the dimensions of the input data.
+            If None is given, then OME-XML metadata will be generated from the data
+            array and any of the following metadata arguments.
+        channel_names: Optional[Union[List[str], List[Optional[List[str]]]]]
+            Lists of strings representing the names of the data channels
+            Default: None
+            If None is given, the list will be generated as a 0-indexed list of strings
+            of the form "Channel:channel_index"
+        physical_pixel_sizes: Optional[Union[types.PhysicalPixelSizes,
+                List[types.PhysicalPixelSizes]]]
+            List of numbers representing the physical pixel sizes in Z, Y, X in microns
+            Default: None
+
+        Note
+        ----
+        This writer can perform chunked writing. When the input array is a dask array or
+        an xarray DataArray backed by dask, data is written one XY plane at a time to
+        conserve memory. This is especially useful when the image will not fit into
+        memory. The underlying write can support further chunking in XY dimensions, so
+        this functionality should be added in the future.
+
+        Raises
+        ------
+        ValueError:
+            Non-local file system URI provided.
+
+        Examples
+        --------
+        Write a TCZYX data set to a tiled OME-Tiff
+
+        >>> image = numpy.ndarray([1, 10, 3, 1024, 2048])
+        ... OmeTiledTiffWriter.save(image, "file.ome.tif")
+
+        If an image is too large to fit into memory, a dask array can be passed in and
+        chunked writing will take place. This will work if each YX plane can fit into
+        memory.
+
+        >>> image = numpy.ndarray([1, 10, 3, 1024, 2048])
+        ... OmeTiledTiffWriter.save(image, "file.ome.tif")
+        """
+        if data.ndim > 5 or data.ndim < 2:
```

Comment:
I am tempted to have you check for bfio import here at the top of the function so that during save if someone tries to use this writer without installing bfio, we can error and give them the standard "please install with extra"